dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/ManageGroupsFeatureIT.java

```diff
@@ -0,0 +1,1687 @@
+            .withParent(groupB)
+            .addMember(eperson)
+            .build();
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        // Verify an ePerson in a sub-subgroup of submitter group doesn't have this feature
+        getClient(token).perform(get("/api/authz/authorizations/search/object?embed=feature&uri="
+            + "http://localhost/api/core/site/" + siteService.findSite(context).getID()))
+            .andExpect(status().isOk())
+            .andExpect(
+                jsonPath("$._embedded.authorizations[?(@._embedded.feature.id=='canManageGroups')]")
+                    .doesNotExist());
+    }
+
+    // Disabled collection configs
+    @Test
+    public void testAdminNoCollectionGroupPermission() throws Exception {
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.policies", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.submitters", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.workflows", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.admin-group", "false");
+
+        String token = getAuthToken(admin.getEmail(), password);
+
+        // Verify the general admin has this feature
+        getClient(token).perform(get("/api/authz/authorizations/search/object?embed=feature&uri="
+            + "http://localhost/api/core/site/" + siteService.findSite(context).getID()))
+            .andExpect(status().isOk())
+            .andExpect(
+                jsonPath("$._embedded.authorizations[?(@._embedded.feature.id=='canManageGroups')]")
+                    .exists());
+    }
+
+    @Test
+    public void testCommunityAdminNoCollectionGroupPermission() throws Exception {
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.policies", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.submitters", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.workflows", "false");
+        configurationService.setProperty(
+            "core.authorization.community-admin.collection.admin-group", "false");
+
+        String token = getAuthToken(communityAdmin.getEmail(), password);
+
+        // Verify the community admin has this feature
+        getClient(token).perform(get("/api/authz/authorizations/search/object?embed=feature&uri="
+            + "http://localhost/api/core/site/" + siteService.findSite(context).getID()))
+            .andExpect(status().isOk())
+            .andExpect(
+                jsonPath("$._embedded.authorizations[?(@._embedded.feature.id=='canManageGroups')]")
+                    .exists());
```

Comment:
Same thing here...if you've *disabled* the ability of Community Admins to manage these groups, shouldn't this return `doesNotExist()`?  (This question applies to all of the `*NoCollectionGroupPermission` tests)

Reply:
Please refer to: https://github.com/DSpace/DSpace/pull/3148#discussion_r580184087